airflow/utils/dag_cycle_tester.py

```diff
@@ -50,8 +50,8 @@ def check_cycle(dag: "DAG") -> None:
     :raises AirflowDagCycleException: If cycle is found in the DAG.
     """
     # default of int is 0 which corresponds to CYCLE_NEW
-    visited = defaultdict(int)
-    path_stack = deque()
+    visited: Dict[str, int] = defaultdict(int)
+    path_stack = deque()  # type: ignore
```

Comment:
Why does this need an ignore? There is `typing.Deque`.

Reply:
Ah cool :). Let me check. I wasn't sure of the type

Reply:
Worked :) :+1: